README.md

```diff
@@ -26,6 +26,42 @@ The monorepo is split into separate packages automatically:
 **Please do not use `contao/contao` in production** but use the split packages
 instead.
 
+## Test setup
+
+To create a Pull Request to this monorepository and test your changes within
+a running Contao 4 application it is easiest to use the [Contao Managed Edition][3].
+Start by installing it in your current directory running
+
+```
+$ composer create-project contao/managed-edition .
+```
+
+Then replace the `require` section so that it does not require the individual bundles
+but this monorepository instead:
+
+```json
+"require": {
+  "php": "^7.1",
+  "contao/contao": "dev-master"
+},
+```
+
+Use `dev-master` for the latest version if you want to introduce new features. Use `dev-<lts-version>` (currently
+`dev-4.4`) if you want to contribute a bugfix.
+To make sure Composer installs the repository from source you might want to delete the `vendor` directory and then
```

Comment:
No longer needed if we would `wget` the `composer.json` file instead of using `create-project`.

Reply:
Why not? You have to adjust the `require` section anyway?

Reply:
Yes, but there will not be a `vendor` directory to delete. 😉